src/templates/fields.md

```diff
@@ -193,7 +193,7 @@ adding a hint, please bear in mind that the easier you make it to answer
 a question in Anki, the less likely you are to remember that question
 when you encounter it in real life. Please have a read about the
 'minimum information principle' on
-<http://www.supermemo.com/articles/20rules.htm> before proceeding.
+<https://supermemo.guru/wiki/20_rules_of_knowledge_formulation> before proceeding.
```

Comment:
This is still the old link

Reply:
Please accept my sincere apologies. I have completely missed that file.

I have fixed it now. Thank you for bringing it to my attention.



HR

Reply:
No problem! Thank you for the changes.